ksp_plugin_adapter/ksp_plugin_adapter.csproj

```diff
@@ -18,22 +17,38 @@
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <OutputPath>..\Debug\GameData\Principia\</OutputPath>
-    <DefineConstants>TRACE;DEBUG;KSP_VERSION_1_7_3</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;KSP_VERSION_1_8_1</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
+    <Prefer32Bit>false</Prefer32Bit>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>full</DebugType>
     <Optimize>true</Optimize>
     <OutputPath>..\Release\GameData\Principia\</OutputPath>
+    <DefineConstants>TRACE;KSP_VERSION_1_8_1</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <DebugSymbols>true</DebugSymbols>
+    <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
+    <Prefer32Bit>false</Prefer32Bit>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```

Comment:
Why not 4.8?  It is [recommended](https://dotnet.microsoft.com/download/dotnet-framework).

Reply:
Mono doesn't mention 4.8 support as of this writing, so let's stay on the safe side https://www.mono-project.com/docs/about-mono/compatibility/.

If we find a 4.8 feature that we want we can try to change this and test whether it works.